frontend/src/app/testQueue/QueueItem.tsx

```diff
@@ -609,10 +609,17 @@ const QueueItem = ({
     // if we want to use a custom date
     if (shouldUseCurrentDateTime()) {
       updateUseCurrentDateTime("false");
+
+      // reset the date fields to their default values
+      const defaultDateString = formatDate(moment().toDate());
+      (document.getElementById(
```

Comment:
Is there a way that we can update the DOM element through the selectedDate variable that is already being used instead of affecting the DOM directly?